src/main/kotlin/no/nav/syfo/dokarkiv/domain/DokarkivRequest.kt

```diff
@@ -26,7 +26,7 @@ data class DokarkivRequest(
             bruker = Bruker.create(id = avsenderMottaker.id, idType = avsenderMottaker.idType),
             dokumenter = dokumenter,
             journalfoerendeEnhet = JOURNALFORENDE_ENHET,
-            journalpostType = "UTGAAENDE",
+            journalpostType = "INNGAAENDE",
```

Comment:
Med forbehold om at de som eier arkivet er enige i at INNGAAENDE er korrekt for dette

Reply:
https://github.com/navikt/dokarkiv/blob/master/journalpost/src/main/java/no/nav/dokarkiv/journalpost/v1/api/opprettjournalpost/OpprettJournalpostRequest.java#L41